apps/api-gateway/src/services/domain_vector_service.py

```
@@ -0,0 +1,275 @@
+            self.embedding_model = SentenceTransformer(model_name)
+            logger.info("DomainVectorService 嵌入模型加载成功", model=model_name)
+        except Exception as e:
+            logger.warning("嵌入模型加载失败，使用哈希向量", error=str(e))
+            self.embedding_model = None
+
+    def _embed(self, text: str) -> List[float]:
+        """生成嵌入向量"""
+        if self.embedding_model:
+            return self.embedding_model.encode(text).tolist()
+        import random
+        random.seed(hashlib.md5(text.encode()).hexdigest())
```

Ruff (0.15.2)

[error] Probable use of insecure hash functions in `hashlib`: `md5`

(S324)